test/cases/runner-debug/runner.debug.spec.js

```diff
@@ -15,6 +15,7 @@ import fs from 'fs-extra';
 import path from 'path';
 import { Runner } from '../../../src/index.js';
 import { fileURLToPath, URL } from 'url';
+import { mock } from 'node:test';
```

Comment:
oh, are we mixing and matching test utilities here?

Reply:
We're not actually mixing test utilities, because this project currently does not have a test mocking framework. Chai is just the assertion library and mocha is just the test runner. Sinon is normally the natural complement to these two for mocking, but you mentioned a preference for the node mocking library (https://github.com/KaiPrince/gallinago/pull/2#issuecomment-3233546481).

So I wouldn't say we're mixing and matching, more so I'm adding our preferred mocking library since we don't currently have one. It made sense to choose the node mocking library over the Sinon library since we want to end up using node test utils in the end anyways and this way we don't have to add Sinon just to get rid of it later.